goinsta.go

```diff
@@ -631,7 +625,8 @@ func (insta *Instagram) UploadPhoto(photo_path string, photo_caption string, upl
 	if err != nil {
 		return response.UploadPhotoResponse{}, err
 	}
-	if _, err = io.Copy(fw, f); err != nil {
```

Comment:
I think `io.Copy` is better choice than `ioutil.ReadAll` for uploading from file , So please do not change `UploadPhoto`